### PR TITLE
Update app icon creation instructions

### PR DESCRIPTION
**Pull request by guilhermedeandrade**
This PR will add new references to the App Icon documentation

### CODE REVIEW

File: docusaurus/docs/welcome/instructions/app-icon.md [hunk 1]
```diff
@@ -85,8 +85,11 @@ How to upload the icon:
 iOS assets can be generated using [App Icon Generator](https://appicon.co/).
 In the "iOS and macOS" section, only `iPhone` and `iPad` should be selected.
 It is possible to either click and browse the image source or simply drag and drop it.
+Note that the file extension must be `.png`.
 After selecting the image, click on the "Generate" button to download the assets.
 
+> ℹ️ An alternative to the App Icon Generator could be [MakeAppIcon](https://makeappicon.com/).
+
 We can also recommend the [Fueled/ios-icon-generator](https://github.com/Fueled/ios-icon-generator) tool, which is a command line interface (CLI) that generates the icons.
```
Comment (acostalima) [inline docusaurus/docs/welcome/instructions/app-icon.md:93]:
```suggestion
**Alternative tools:**
- [MakeAppIcon](https://makeappicon.com/): a web generator by [Skygear](https://skygear.io/)
- [`ios-icon-generator`](https://github.com/Fueled/ios-icon-generator): a CLI generator by [Fueled](https://fueled.com/)
```

**Comment by guilhermedeandrade** [comment]
> ```diff
- This way, it becomes makes easier to identify the size of each icon and their purpose simply by glancing at its name.
+ This way, it becomes easier to identify the size of each icon and their purpose simply by glancing at its name.
> ```

I think we should remove the "makes", right? It seems wrong

File: docusaurus/docs/welcome/instructions/app-icon.md [hunk 2]
```diff
@@ -85,8 +85,11 @@ How to upload the icon:
 iOS assets can be generated using [App Icon Generator](https://appicon.co/).
 In the "iOS and macOS" section, only `iPhone` and `iPad` should be selected.
 It is possible to either click and browse the image source or simply drag and drop it.
+Note that the file extension must be `.png`.
```
[the remark this answers is not in the document]
Comment (guilhermedeandrade) [reply docusaurus/docs/welcome/instructions/app-icon.md:88]: Done

File: docusaurus/docs/welcome/instructions/app-icon.md (hunk 1 above)
Comment (guilhermedeandrade) [reply docusaurus/docs/welcome/instructions/app-icon.md:95]: Done

**Comment by guilhermedeandrade** [comment]
> ```diff
> - This way, it becomes makes easier to identify the size of each icon and their purpose simply by glancing at its name.
> + This way, it becomes easier to identify the size of each icon and their purpose simply by glancing at its name.
> ```
> 
> I think we should remove the "makes", right? It seems wrong

I went ahead and made the change, since I'm pretty sure this was badly written by me

**Pull request merged**